rlp/benches/rlp.rs

```diff
@@ -14,11 +14,10 @@
 
 #![feature(test)]
 
-extern crate ethereum_types;
-extern crate rlp;
+// TODO: get rid of this one fine day: https://doc.rust-lang.org/nightly/edition-guide/rust-2018/module-system/path-clarity.html#an-exception
```

Comment:
Let's create an issue to convert benches to criterion instead.

Reply:
https://github.com/paritytech/parity-common/issues/171